lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -43,9 +43,15 @@ def self.run(params)
         if release_id.nil?
           return
```

Comment:
When is release_id ever `nil`? Should we be failing loudly if it is?

Reply:
No this is when we try to many times and it doesn't upload successfully. This is for a graceful shutdown to give the user the fastlane summary at the end. See  https://google-fabric.slack.com/archives/C014P4EQMR7/p1593629261220000  for screenshots of the way it would shutdown with a return vs exit. A crash seemed like too much, but we can discuss if it is appropriate.